puara_gestures.h

```diff
@@ -35,6 +36,14 @@ class PuaraGestures {
         float jabY;
         float jabZ;
         int jabThreshold = 10;
+        void updateJabShake();
+        // Orientation
+        const float DECLINATION = -14.14; // Declination at Montreal on 2020-03-12
```

Comment:
Could be useful to expose this parameter, in case someone tries to play a T-Stick in China......

We can then add it to the T-Stick settings tab/JSON